Update README.md for 0.7 (#4352)
quickwit-oss/quickwit: README.md

</p>

<h2 align="center">

> Cloud-native search engine for Observability (logs, traces, and soon metrics!). An open-source alternative to DataDog, Elasticsearch, etc.

Reviewer: ```suggestion
Cloud-native search engine for observability (logs, traces, and soon metrics!). An open-source alternative to Datadog, Elasticsearch,  Loki, and Tempo.
```

</h4>
<br/>

<b>We just released Quickwit 0.7! Read the [blog post](https://quickwit.io/blog/quickwit-0.7) to learn about the latest powerful features!</b>

> ### **Quickwit is the fastest search engine on cloud storage. It's the perfect fit for observability use cases.**

Reviewer: ```suggestion
### **Quickwit is the fastest search engine on cloud storage. It's the perfect fit for observability use cases**
```

- [Log management](https://quickwit.io/docs/log-management/overview)

> - [Distributed traces](https://quickwit.io/docs/distributed-tracing/overview)

Reviewer: ```suggestion
- [Distributed tracing](https://quickwit.io/docs/distributed-tracing/overview)
```

<br/>

# 💡 Features

- Full-text search and aggregation queries

> - Elasticsearch compatible API

Reviewer: ```suggestion
- Elasticsearch-compatible API, use Quickwit with any Elasticsearch or OpenSearch client
```

- [Jaeger-native](https://quickwit.io/docs/distributed-tracing/plug-quickwit-to-jaeger)
- OTEL-native for [logs](https://quickwit.io/docs/log-management/overview) and [traces](https://quickwit.io/docs/distributed-tracing/overview)
- [Schemaless](https://quickwit.io/docs/guides/schemaless) or strict schema indexing
- Schemaless analytics

> - Sub-second search on cloud storage (Amazon S3, Azure Blob Storage, …)

Reviewer: ```suggestion
- Sub-second search on cloud storage (Amazon S3, Azure Blob Storage, Google Cloud Storage, …)
```
